.github/workflows/ci.yml

```diff
@@ -43,47 +46,75 @@ jobs:
     name: Project Checks
     runs-on: ubuntu-latest
     timeout-minutes: 5
-
     steps:
-      - uses: actions/setup-go@v3
+      -
+        uses: actions/setup-go@v3
         with:
           go-version: 1.17
-      - uses: actions/checkout@v3
+      -
+        uses: actions/checkout@v3
         with:
           path: src/github.com/uor-framework/client
-      - name: verify go modules and vendor directory
+      -
+        name: verify go modules and vendor directory
         run: |
           make vendor
         working-directory: src/github.com/uor-framework/client
-      - name: running unit tests
+      -
+        name: running unit tests
         run: |
           make test-unit
         working-directory: src/github.com/uor-framework/client
-      - name: running sanity checks
+      -
+        name: running sanity checks
         run: |
           make sanity
         working-directory: src/github.com/uor-framework/client
 
-    # Release
+  # Release
   release:  
     needs: [linters, project]
     runs-on: ubuntu-latest
     timeout-minutes: 10
-
     if: startsWith(github.ref, 'refs/tags/')
     steps:
-    - uses: actions/checkout@v3
+    -
+      name: Checkout Repo
+      uses: actions/checkout@v3
+      id: clone
       with:
           fetch-depth: 0
-    - uses: actions/setup-go@v3
+    -
+      name: Install Cosign
+      uses: sigstore/cosign-installer@v2.4.1
+      id: cosign
+    -
+      name: Install SBOM
+      uses: anchore/sbom-action/download-syft@v0.11.0
+      id: sbom
+    -
+      name: Install QEMU
+      uses: docker/setup-qemu-action@v2
+      id: qemu
+    -
+      name: Install QEMU
+      uses: docker/setup-buildx-action@v2
+      id: buildx
+      with:
+        install: true
+    -
+      name: Install Go
+      uses: actions/setup-go@v3
+      id: go
       with:
         go-version: 1.17
-    - name: release with goreleaser
+    -
+      name: Run GoReleaser
       id: goreleaser
       uses: goreleaser/goreleaser-action@v2
       with:
-        version: latest
+        distribution: goreleaser
```

Comment:
Receiving errors when testing 

https://github.com/sabre1041/uor-client/runs/7400218665?check_suite_focus=true

Reply:
I'm not super familiar, but it looks like the build arg `TARGETARCH` isn't defined for this stage: https://github.com/sabre1041/uor-client/blob/396a922199308321b674a2dbf25ad2a37730956c/Containerfile#L16. 

So just add that variable to wherever the build is happening! (I can look more closely after dinner, kind of curious how this Containerfile works)!

Reply:
I'm thinking either you can add (or hardcode it if it's just one target for in the container) in this section here, which is what I think drives the buildx build during the gorelease action?

https://github.com/sabre1041/uor-client/blob/396a922199308321b674a2dbf25ad2a37730956c/.goreleaser.yaml#L81-L95

or don't do that, but set a default in the Containerfile for it.

Reply:
The variable [`TARGETARCH`](https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope) ([hyperlink](https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope)) is a global variable in docker buildx builds which is sourced from the explicit or implicit platform flag `--platform=linux/amd64`. 

I am relying on this behavior as groundwork to support multi-arch container builds in subsequent PRs from this single Containerfile.

This particular Containerfile design is written to achieve a minimalist / nearly distroless container end result while still being built exclusively from UBI rpm sources.

I seem to have inadvertently broken the Containerfile by moving the `ARG TARGETARCH` line above the second `FROM` line while doing some 'house keeping' in the containerfile.

Reply:
Gotcha! And totally OK, those kinds of things happen once in a while. I’ve actually never worked with the Go release action or seen a containerfile so I’m enjoying learning about it! I should give this a whirl for some of my Go projects.

Reply:
goreleaser is new to me too, it's been a fun puzzle to play with :)